lib/lyber_core/boot.rb

```diff
@@ -60,7 +60,7 @@ def boot_dsa
 
       Dor::Services::Client.configure(url: Settings.dor_services.url,
                                       token: Settings.dor_services.token,
-                                      enable_get_retries: true)
```

Comment:
is the dsc change why this is now set to false?   from the PR title ... don't get it.   Probably have the same reaction to the commit message.

Reply:
Actually, true is now the default so no reason to explicitly set here.